docs/getting-started.md

```diff
@@ -105,3 +110,35 @@ Or to start the auto-restarting development server:
 ```sh
 make start-dev
 ```
+
+Run Docker for development
+----------------------
+
+The `Dockerfile.dev` and `docker-compose.yml` file at the root of the project work together to start the
+development server with a MySQL database.
+
+Note: so far this has been built with a bias for ARM64/Apple Silicon machines. 
+
+Prerequisites:
+
+- [Docker Desktop](https://docs.docker.com/desktop/) or CLI
```

Comment:
I think this is only something mac users need? Maybe we can just say that users need to have docker running.

Reply:
It's just a GUI, and there's versions for windows, mac, linux.

Reply:
Yeah but it's closed source and Linux definitely don't need it.

On a mac a tool like this is required I think because docker can't run natively, but there's alternatives like: https://rancherdesktop.io/

Maybe it just needs to be qualified like. "- Docker (on Mac you may need a tool like Docker Desktop or Rancher)".